submodules/repositories/events_repository.py

```diff
@@ -13,14 +13,27 @@ def events(self):
         return self.__service.service().events()
 
     def next_event(self) -> Dict[str, Any]:
-        now = datetime.datetime.utcnow().isoformat() + 'Z'
-        print(f'Getting event from: {now.split()}')
-        request = self.events().list(calendarId=self.__calendar_id, timeMin=now,
+        # now = datetime.datetime.utcnow().isoformat() + 'Z'
+        now = datetime.datetime.utcnow()
+        next_class = now + datetime.timedelta(days = 4)
+        print(f'Getting event from: {now.isoformat().split()}')
+        request = self.events().list(calendarId=self.__calendar_id, timeMin=now.isoformat()+"Z", timeMax = next_class.isoformat()+"Z", 
                                      maxResults=1, singleEvents=True, orderBy='startTime')
```

Comment:
Por ahi muy quisquilloso, pero creo que estaria bueno sacar los timestamps de la funcion, mas que nada por el paso extra del agregarles el "Z" que puede ser perdido en una leida rapida.

```suggestion
        now_iso_string = now.isoformat() + "Z"
        next_class_iso_string = next_class.isoformat() + "Z"
        
        request = self.events().list(
            calendarId=self.__calendar_id
            timeMin=now_iso_string,
            timeMax=next_class_iso_string, 
            maxResults=1,
            singleEvents=True,
            orderBy='startTime'
        )
```

Reply:
Estoy de acuerdo, ya esta cambiado